### PR TITLE
Added Update User Info

### PR DESCRIPTION
**Pull request by zhairanicolepacheco**

### CODE REVIEW

File: user-management-application/src/main/java/com/user/management/Main.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        System.out.println("1. Update User Information");
+        System.out.println("0. Exit");
+    }
+
+    private static void updateUserInformation() {
+        try {
+            System.out.print("Enter User-ID: ");
+            int userId = scanner.nextInt();
+
+            if (userId <= 0) {
```
Comment (JustineDaveMagnaye) [inline user-management-application/src/main/java/com/user/management/Main.java:59]: UserId can be 0 above

File: user-management-application/src/main/java/com/user/management/data/user/dao/impl/UserDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import com.user.management.app.model.user.User;
+import com.user.management.data.connection.ConnectionHelper;
+import com.user.management.data.user.dao.UserDao;
+
+import java.sql.*;
+
+public class UserDaoImpl implements UserDao {
+
+    @Override
+    public User getUserById(int id) {
```
Comment (JustineDaveMagnaye) [inline user-management-application/src/main/java/com/user/management/data/user/dao/impl/UserDaoImpl.java:18]: Combine the ExtractResultSet method to the getUserById method

File: user-management-application/src/main/java/com/user/management/data/user/dao/impl/UserDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            stmt.setInt(5, user.getId());
+            int affectedRows = stmt.executeUpdate();
+            return affectedRows > 0;
+        } catch (SQLException ex) {
+            ex.printStackTrace();
+            return false;
+        }
+    }
+
+    private User extractUserFromResultSet(ResultSet rs) throws SQLException {
```
Comment (JustineDaveMagnaye) [inline user-management-application/src/main/java/com/user/management/data/user/dao/impl/UserDaoImpl.java:52]: Combine the ExtractResultSet method to the getUserById method

File: user-management-application/src/main/java/com/user/management/data/connection/ConnectionHelper.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+package com.user.management.data.connection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+
+public class ConnectionHelper {
+    public static final String URL = "jdbc:oracle:thin:@localhost:1521:rogate"; //SSID na gamit, instance ng database
+    public static final String ORACLE_DRIVER = "oracle.jdbc.driver.OracleDriver";
+    public static final String username = "system"; //username
```
Comment (RoCS2024) [inline user-management-application/src/main/java/com/user/management/data/connection/ConnectionHelper.java:15]: use a proper naming convention 
`public static final String USERNAME`

File: user-management-application/src/main/java/com/user/management/data/connection/ConnectionHelper.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    public static final String password = "Changeme0"; //password
```
Comment (RoCS2024) [inline user-management-application/src/main/java/com/user/management/data/connection/ConnectionHelper.java:16]: fix naming convention

**Review by rosellemarithe — APPROVED** [review]

**Review by rosellemarithe — APPROVED** [review]

**Review by rosellemarithe — APPROVED** [review]

**Review by daiahnadine — APPROVED** [review]

**Review by JustineDaveMagnaye — APPROVED** [review]

**Pull request merged**